src/clj/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_service.clj

```diff
@@ -12,7 +12,7 @@
 
   (init
     [this context]
-    {:watchers (atom [])})
+    (assoc context :watchers (atom [])))
```

Comment:
Was this change strictly necessary or did it just seem like a good idea?

Isn't `(= context {})` at this point?